### ISSUE TITLE
asPercent and divideSeries doesn't support multiple series as total?

### ISSUE THREAD

**Issue by Dieterbe** [go-graphite/carbonapi#213]
per http://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.asPercent 

> The total parameter may be a single series, reference the same number of series as seriesList or a numeric value.

however seems like the code rejects the 2nd parameter if it returns multiple series:

```
			total, err := getSeriesArg(e.args[1], from, until, values)
			if err != nil {
				return nil, err
			}
			if len(total) != 1 {
				return nil, ErrWildcardNotAllowed
			}
```

any particular reason why? PS I'm not sure what's supposed to happen if it's multiple series.

**Comment by arodland** [comment]
It's pairwise — the series in seriesList are matched one-for-one with the series in total. Basically like divideSeries except with results 100 times as big. carbonapi doesn't handle it for divideSeries either, though, so can't just lift the solution from there :)

**Comment by Dieterbe** [comment]
> Basically like divideSeries except with results 100 times as big. carbonapi doesn't handle it for divideSeries either, 

seems to me like graphite's divideSeries only accepts a single series as divisor. see http://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.divideSeries if it were a series it would use seriesLists as second argument.
maybe you got it confused with divideSeriesLists ?

the code for divideSeries also says:
```
  if len(divisorSeries) > 1:
    raise ValueError("divideSeries second argument must reference exactly 1 series (got {0})".format(len(divisorSeries)))
```

**Comment by Civil** [comment]
Am I correct that current master now have support for divideSeriesLists and also have graphite-compatible asPercent implementation?

**Comment by deniszh** [comment]
Yes,
https://github.com/go-graphite/carbonapi/commit/ba333a64c26af9a0423751f7ceba7392a5715319 - divideSeriesLists
https://github.com/go-graphite/carbonapi/commit/0b833c2e58560fe73ba45318238651b37962cf25 - Make asPercent compatible with Graphite

**Comment by sparr** [comment]
I recently encountered this.

asPercent(foo.*.bar, foo.*.blah) works in graphite, not in go-graphite.

**Comment by ibuclaw** [comment]
asPercent should work starting from #257

**Comment by RudW0lf** [comment]
Is it fixed?

**Comment by Civil** [comment]
This should be fixed in recent 0.9.x releases
